src/api/app/controllers/person_controller.rb

```diff
@@ -280,7 +280,7 @@ def command_token
     if params[:project] || params[:package]
       pkg = Package.get_by_project_and_name( params[:project], params[:package] )
     end
-    @token = Token.create( user: user, package: pkg )
+    @token = Token::Service.create( user: user, package: pkg )
```

Comment:
Is used by the API just for running services... see what Adrian said about the usecases